releasenotes-dca/notes/orchestrator-data-race-informer-45a137b6424ae398.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG-DCA.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+issues:
+  - |
+    Fix major data races in orchestrator check Kubernetes resource collection.
```

Let me know if that change the meaning.
```suggestion
    Fix major data races in the orchestrator check for the Kubernetes resource collection.
```